### PR TITLE
Fix/slc start ux improve

### PR DESCRIPTION
**Pull request by jtary**
Some small changes into fix the major UX issues introduced with slc-start.

connected to #1096

### CODE REVIEW

File: process-manager/process-manager.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ ProcessManager.prototype.start = function(cb) {
       return cb(err);
     }
 
+    process.env.PORT = '3001';
     pm.process = spawn(process.execPath, args, {
```
Comment (sam-github) [inline process-manager/process-manager.js]: there is an env: option to spawn(), use that: make sure to `util._extend(process.env, {PORT: 3001})`

**Comment by jtary** [comment]
@sam-github PTAL

**Comment by sam-github** [comment]
LGTM

**Comment by seanbrookes** [comment]
moving this to done as I'm having trouble going backwards with my current mesh models/client code

**Pull request merged**